libs/styles/styles.css

```diff
@@ -496,12 +496,45 @@ div[class*='-up'] .con-block.has-bg.xxl-spacing { padding: var(--spacing-xxl); }
 div[class*='-up'] .con-block.has-bg.xxxl-spacing { padding: var(--spacing-xxxl); }
 div[class*='-up'] .con-block.has-bg.no-spacing { padding: 0; }
 
+.con-block .foreground { position: relative; }
+
 .con-block.max-width-6 .foreground { max-width: 600px; }
 .con-block.max-width-8 .foreground { max-width: 800px; }
 .con-block.max-width-10 .foreground { max-width: 1000px; }
 .con-block.max-width-12 .foreground { max-width: 1200px; }
 .con-block.max-width-auto .foreground { max-width: unset; }
 
+.con-block .background {
+  overflow: hidden;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
+
+.con-block .background>div { height: 100%; }
+
+.con-block .background img,
+.con-block .background video {
+  object-fit: cover;
+  min-height: 700px;
```

Comment:
I'm seeing an issue w/ this height being set all the time. 

```suggestion
```
If we need a min height on .background assets, aka `.marquee.large`, we should add that in the block specific css. 
This is causing some assets to get clipped in smaller height views. 

<img width="1368" alt="Screen Shot 2023-09-19 at 3 45 41 PM" src="https://github.com/adobecom/milo/assets/2086146/992d359e-317f-422f-8489-c9b8e687ac60">
<img width="1507" alt="Screen Shot 2023-09-19 at 3 45 56 PM" src="https://github.com/adobecom/milo/assets/2086146/aaffb38b-7fed-42a0-8e1e-3a916950e7c8">